src/zzz_od/application/charge_plan/charge_plan_app.py

```
@@ -57,10 +60,67 @@ def start_charge_plan(self) -> OperationRoundResult:
+    @operation_node(name='查看双倍活动')
+    def check_double_reward_event(self):
+        """
+        实战模拟室的每日双倍总数都是5次, 刷盘子的2次暂不考虑. 理由:
+        1. 如果没抽角色, 体力没事干的时候就去屯金盘收益最高, 此时体力计划能顺便覆盖盘子双倍
+        2. 双倍盘子活动出现的时机一般在卡池开了之后几天, 此时如果抽了角色也差不多养好了要刷盘子了
+        3. 双倍基础材料活动都是在版本末期, 没事干的时候整一出双倍材料, 总不能没事干的时候一直刷基础材料吧.
+           这个活动就显得很鸡肋, 像是专门给预抽卡的人群(比如氪佬)设计的
+        """
+
+        # 跳转双倍活动界面
+        result1 = self.round_by_goto_screen(screen_name=f'快捷手册-训练')
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)